LocalSettings.php

```diff
@@ -5708,6 +5709,19 @@
 		'fbwikiwiki' => array('visualeditor'),
 		'spiralwiki' => array('visualeditor'),
 	),
+	
+	// Protect site config
+	'wgProtectSiteLimit' => array(
+		'default' => '1 week',
+		'tnoteswiki' => 'indefinite',
+	),
+	'wgProtectSiteDefaultTimeout' => array(
+		'default' => '1 hour',
+		'tnoteswiki' => '2 hours',
+	),
+	'wgProtectSiteExempt' => array(
```

Comment:
Unless a wiki wants a different value than the default, I'm not sure why we should be listing the default. That means we'd have to list all the default values for all the extensions.